{{cookiecutter.project_slug}}/_/deployment/database/base/deployment.yaml

```diff
@@ -28,7 +28,8 @@ spec:
             - /bin/sh
             - -i
             - -c
-            - psql -h 127.0.0.1 -U $POSTGRESQL_USER -q -d $POSTGRESQL_DATABASE -c
+            - PGPASSWORD=$POSTGRESQL_PASSWORD 
+              psql -h 127.0.0.1 -U $POSTGRESQL_USERNAME -q -d $POSTGRESQL_DATABASE -c
```

Comment:
I don't understand to role of `PGPASSWORD`. There is no reference to it [in the image documentation](https://hub.docker.com/r/bitnami/postgresql) on Docker Hub.

Reply:
This was an underdocumented feature that we found when troubleshooting (with @srueg ).  The bitnami postgres image reads the password from this environment variable and there's no variant to put the password on the command-line.  So, either we set it in the same command, or it waits forever for the password.